tests/mock_network/node_ageing.rs

```diff
@@ -150,6 +151,68 @@ fn relocate_during_split() {
     })
 }
 
+// Test that during startup phase all churn events cause age increments.
+#[test]
+fn startup_phase() {
+    let env = Environment::new(NETWORK_PARAMS);
+    let mut nodes = vec![];
+
+    // Only the first `safe_section_size - 1` adds cause age increments, the rest does not.
+    for i in 0..(env.safe_section_size() + 1) {
+        trace!("add node {}", i);
+        add_node_to_section(&env, &mut nodes, &Prefix::default());
+
+        poll_until(&env, &mut nodes, |nodes| {
+            node_joined(nodes, nodes.len() - 1)
+        });
+
+        poll_until(&env, &mut nodes, |nodes| {
+            check_root_section_age_counters_after_only_adds(&env, nodes)
+        })
+    }
+}
+
+// Verify that the age counters of all the nodes in the root section are as expected assuming we
```

Comment:
... are as expected, assuming we were ...
or 
have a bracket around `assuming we were ... not removing`